src/edgeProxy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "edgeproxy",
+    "version": "0.0.0",
+    "private": true,
+    "scripts": {
+        "buildLib": "rm -rf public/lib; mkdir public/lib; cp -rf node_modules/infusion public/lib/infusion",
```

Comment:
The ; syntax is only supported on bash and && should be used instead for cross-platform portability